element_deeplabcut/dlc.py

```diff
@@ -354,68 +305,66 @@ class BodyPart(dj.Part):
         """
 
     @classmethod
-    def insert_new_model(cls, session_key, config_paramset_idx, model_name,
-                         model_description='', training_id=None,
-                         body_part_descriptions=None):
-        """ Add new model to DataJoint framework

```

Comment:
Do we want docstrings in these class methods or no? I go through and copy them over from my previous draft